src/main/java/io/kontur/disasterninja/service/layers/providers/BivariateLayerProvider.java

```diff
@@ -154,7 +154,17 @@ private Legend bivariateLegendFromOverlay(BivariateLayerLegendQuery.Overlay over
                 xAxis.setSteps(steps);
             }
             xAxis.setLabel(x.label());
-            xAxis.setQuotient(x.quotient());
```

Comment:
I guess we have to keep old `quotient` as well as new `quotients` together for some time to support backwards compatibility for FE. Otherwise something can be broken.